projects/addon-doc/src/components/example/example.style.less

```diff
@@ -66,7 +66,7 @@
     padding: 2rem;
     overflow-x: auto;
 
-    @media @mobile-m {
+    :host-context(tui-root._mobile) {
```

Comment:
@vladimirpotekhin doesn't this change e2e break?

Reply:
I don't think so, we don't run e2e on mobile

Reply:
sorry I see some components with mobile width, but I don't know how it can breaks e2e, behaviour should be the same. Let's try to return media to check

Reply:
fixed